apidef/oas/schema/x-tyk-api-gateway.json

```diff
@@ -2062,10 +2072,24 @@
                 "null"
               ]
             },
-            "headerName": {
-              "type": "string"
+            "header": {
```

Comment:
this could be of type `X-Tyk-AuthSource`

Reply:
we could use it in here as it's not necessarely an issue in the oas side, let me know after you agree with the above comment